templates/extensions/foundry/packages/foundry/script/Deploy.s.sol.template.mjs

```diff
@@ -11,17 +14,23 @@ contract DeployScript is ScaffoldETHDeploy {
         uint256 deployerPrivateKey = setupLocalhostEnv();
         if (deployerPrivateKey == 0) {
             revert InvalidPrivateKey(
-                "You don't have a deployer account. Make sure you have set DEPLOYER_PRIVATE_KEY in .env or use `yarn generate` to generate a new random account"
+                "You don't have a deployer account. Make sure you have set DEPLOYER_PRIVATE_KEY in .env or use \`yarn generate\` to generate a new random account"
             );
         }
         vm.startBroadcast(deployerPrivateKey);
-        YourContract yourContract =
-            new YourContract(vm.addr(deployerPrivateKey));
+
+        YourContract yourContract = new YourContract(
+            vm.addr(deployerPrivateKey)
+        );
         console.logString(
             string.concat(
-                "YourContract deployed at: ", vm.toString(address(yourContract))
+                "YourContract deployed at: ",
+                vm.toString(address(yourContract))
             )
         );
+
+        ${deploymentsLogic.filter(Boolean).join("\n")}
```

Comment:
should it be moved below `vm.stopBroadcast()`? See first error https://github.com/scaffold-eth/create-eth/pull/43#issuecomment-2145651951

Reply:
Ohh yes fixed at https://github.com/scaffold-eth/create-eth/pull/43/commits/0c109be9ad2895222a09602487b08f540c627d5c thanks !